source/index.html.erb

```diff
@@ -54,13 +54,13 @@ title: OKD - The Origin Community Distribution of Kubernetes that powers Red Hat
             <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
               <h2>Run OKD in a Container</h2>
               <p>
-                Want to get started quickly and easily? Just <a href="https://docs.okd.io/latest/getting_started/administrators.html#running-in-a-docker-container">run OKD in a Docker container</a> by itself.
+                Want to get started quickly and easily? Just <a href="https://docs.okd.io/latest/getting_started/administrators.html#running-in-a-docker-container">run OKD in a  container</a> by itself.
               </p>
```

Comment:
Nit: looks like a doubled stace here.  Doesn't matter in HTML, so maybe not worth cleaning up, but I thought I'd point it out in case anyone wanted to clean it up ;)

Reply:
👍 Fixed in #74